From 92e4c52a45016838fbfaea0fc5b92c58278d2e46 Mon Sep 17 00:00:00 2001
From: Kelan Ige <113405702+kelanige@users.noreply.github.com>
Date: Mon, 24 Jul 2023 18:47:52 -0700
Subject: [PATCH] Only build scull misc-modules

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7b08c49..c9cd253 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,9 @@
 
-SUBDIRS =  misc-progs misc-modules \
+#SUBDIRS =  misc-progs misc-modules \
            skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
 	   pci usb\
 	   sbull snull short
+SUBDIRS = scull misc-modules
 
 all: subdirs
 
